.gitignore

```diff
@@ -2,4 +2,5 @@
 .env
 .env.local
 node_modules/
-dev-db.sqlite
\ No newline at end of file
+dev-db.sqlite
+dist/
```

Comment:
I forgot to mention you have to `git rm dist/bundle.js` as well and commit its removal since the ignore config only affects file adds in the future.